src/material/autocomplete/autocomplete.spec.ts

```diff
@@ -3371,6 +3371,32 @@ describe('MatAutocomplete', () => {
 
     expect(fixture.componentInstance.trigger.panelOpen).toBe(true);
   });
+
+  it('should emit from `autocomplete.closed` after click outside inside the NgZone', fakeAsync(() => {
+    const inZoneSpy = jasmine.createSpy('in zone spy');
+
+    const fixture = createComponent(SimpleAutocomplete, [
+      {provide: NgZone, useFactory: () => new NgZone({enableLongStackTrace: false})},
```

Comment:
Why is this necessary?

Reply:
> Why is this necessary?

MockZone always returns false for isInAngularZone(), it doesn't depend on running in zone.run or outside

And when common NgZone is used then it displays in correct way.

Otherwise some another test should be implemented, I think this is excessively

I've not found any solution how to pass real NgZone into the function

Reply:
ok, I see. I think that you should be able to at least reduce the `providers` array to `[NgZone]` or `[{provide: NgZone, useClass: NgZone}]`.

Reply:
> ok, I see. I think that you should be able to at least reduce the `providers` array to `[NgZone]` or `[{provide: NgZone, useClass: NgZone}]`.

No it is impossible because of [NgZone constructor expects zoneOptions object](https://github.com/angular/angular/blob/3e46a426c47bf6589952c7f2a17d914dd6f38dfe/packages/core/src/application_ref.ts#L201)

Reply:
Makes sense.